ts/store/reducers/entities/services/index.ts

```diff
@@ -26,11 +38,158 @@ export type ServicesState = Readonly<{
   readState: ReadStateByServicesId;
 }>;
 
+export type ServicesSectionState = Readonly<{
+  organizationName: string;
+  organizationFiscalCode: string;
+  data: ReadonlyArray<pot.Pot<ServicePublic, Error>>;
+}>;
+
 const reducer = combineReducers<ServicesState, Action>({
   byId: servicesByIdReducer,
   byOrgFiscalCode: serviceIdsByOrganizationFiscalCodeReducer,
   visible: visibleServicesReducer,
   readState: readServicesByIdReducer
 });
 
+// Selectors
+export const servicesSelector = (state: GlobalState) => state.entities.services;
+
+// Check if the passed service is local or national throught data included into the service metadata
+const getLocalizedServices = (
+  service: pot.Pot<ServicePublic, Error>,
+  servicesMetadataById: ServiceMetadataById,
+  localization?: ScopeEnum
+) => {
+  if (localization === undefined) {
+    return true;
+  }
+
+  const id = pot.isSome(service) ? service.value.service_id : undefined;
+  if (id) {
```

Comment:
if **id** is undefined should we return false?

Reply:
updated within https://github.com/teamdigitale/io-app/pull/1269/commits/492815f0be07f5cc8c472ced96be09ea8e185d87